ruby/app.rb

```diff
@@ -176,36 +204,56 @@
 # rubocop:disable Metrics/BlockLength
 get '/api/assets' do
   begin
+    options = {
+      client_report_id: '123',
+      webhook: 'https://www.example.com',
+      user: {
+        client_user_id: '789',
+        first_name: 'Jane',
+        middle_name: 'Leah',
+        last_name: 'Doe',
+        ssn: '123-45-6789',
+        phone_number: '(555) 123-4567',
+        email: 'jane.doe@example.com'
+      }
+    }
```

Comment:
why was this not required in the old code?

Reply:
none of the options are necessarily required.  This was added in the Ruby test in the generated clibs, so I left it in.